### DISCUSSION TITLE
When does a newly added language appear on live site after PR merged?

### DISCUSSION THREAD

**Discussion by satnaing** [monkeytypegame/monkeytype#3478] [Questions] [answered]
I've added a new language (_Burmese_) and my pull request got merged. However, I can't find _Burmese_ typing option on the site. 
Is there any other process before appearing on the website?
Or maybe my newly added language has some problem?

**Comment by Miodec** [comment, marked as answer]
There is no set deployment schedule. Sometimes it appears very quickly, sometimes after a few days. (ill deploy now so you should see it soon)

**Reply by satnaing** [reply]
Yes, thank you!

**Comment by satnaing** [comment]
BTW
I found one more issue. The language is not displaying properly.
<img width="905" alt="Screen Shot 2022-08-30 at 10 41 50 PM" src="https://user-images.githubusercontent.com/53733092/187487430-de129be4-1526-4fc3-abba-df1c90062f00.png">

In Burmese, even though we have to type like this
ဖ + ခ + င + ်
It should display like this
ဖခင်

Current issue is that all words are being displayed how it should be typed; not how it should be displayed.
Should I set "ligatures": true?

**Reply by satnaing** [reply]
Yes
I set this project up in my local and turns out I need to set ligatures to true.
I'll make another pull request for this. Thz.
